lib/Ushur.js

```diff
@@ -126,18 +128,22 @@ Ushur.prototype.getResponses = function (params = {}) {
   var _this = this;
   var payload = {
     requestId: params.requestId,
-    cmd: params.requestId ? 'responseForReqId' : undefined,
+    cmd: params.requestId ? "responseForReqId" : params.cmd,
```

Comment:
what is this change?

Reply:
earlier cmd value was ignored(used **undefined** instead) even though it was sent in the request.